frame/support/src/traits.rs

```diff
@@ -1733,11 +1733,10 @@ pub trait Instance: 'static {
 
 /// An instance of a storage.
 ///
-/// It is required the the couple `(PalletInfo::name<Pallet>(), STORAGE_PREFIX)` is unique.
+/// It is required the the couple `(pallet_prefix(), STORAGE_PREFIX)` is unique.
```

Comment:
Where do we use this tuple?

Reply:
we don't use it as a tuple, but just all storages must use different pallet_prefix and storage_prefix otherwise they would be after the same key `twox_128(pallet_prefix)++twox_128(storage_prefix)`.

But the doc is not good, I updated